tests/trestle/tasks/osco_test.py

```diff
@@ -0,0 +1,141 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OSCO transformation tests."""
+
+import base64
+import bz2
+import json
+import pathlib
+import uuid
+from unittest.mock import patch
+
+from trestle.utils import osco
+
+import yaml
+
+stem_in = pathlib.Path('tests/data/tasks/osco/input')
+stem_in_no_metadata = pathlib.Path('tests/data/tasks/osco/input-no-metadata')
+stem_out = pathlib.Path('tests/data/tasks/osco/output')
+
+mock_uuid4_value = uuid.UUID('d4cf7a88-cea7-4667-9924-8d278dc843df')
+
+
+def test_function_osco_get_observations(tmpdir):
```

Comment:
if you replace tmpdir with tmp_path it is the same thing but a pathlib object - which will remove the need for line 45.

Reply:
Fixed.